tests/config/test_configtypes.py

```diff
@@ -1124,15 +1151,16 @@ class FontTests(unittest.TestCase):
     }
     INVALID = ['green "Foobar Neue"', 'italic green "Foobar Neue"',
                'bold bold "Foobar Neue"', 'bold italic "Foobar Neue"'
-               'bold', '10pt 20px "Foobar Neue"']
+                                          'bold', '10pt 20px "Foobar Neue"']
```

Comment:
I think that indentation broke because the `'bold'` at the bottom is actually part of the `'bold italic "Foobar Neue"'` string in the line above it, since there is no colon between them which means Python joins both strings. Is this intended @The-Compiler? Either way, changing the string above to `'bold italic "Foobar Neue" bold'` or leaving out the trailing `bold` the test passes.